### DISCUSSION TITLE
work offline and detect

### DISCUSSION THREAD

**Discussion by kigy1** [BlafKing/sd-civitai-browser-plus#153] [Ideas]
hello, thank you for your work and your extension

1-i would if i can browse the info and image for installed model and lora etc.(**offline**)
2-the extension can't detect the model that not download with it, only one way to detect is to press the button near edit meta (**need one button to recognize all model and lora etc. in the file and download info and image**)

thanks again..

**Comment by BlafKing** [comment]
Thanks for posting, about your suggestions:

1. Unfortunately it won't be possible to pull the model info offline, since the CivitAI API is used to display any info about the model, which is only possible with an internet connection.

2. A way to make the extension detect models that weren't downloaded with the tool is to go to the `Update Models` tab, then select `All` models, and then `Load all installed models`, this will generate a hash and model ID for each model that was downloaded externally so that it can be used in the extension!
(Make sure to keep the `One-Time Hash Generation for externally downloaded models` toggled on)

**Reply by kigy1** [reply]
Thanks for the response, perhaps in the future if a technique appears that can takes data from the html file, jason and image offline that downloaded from extension
